src/features/welcomePage/index.ts

```diff
@@ -0,0 +1,309 @@
+
+              <div id="system-readiness" class="statusDisplay"></div>
+
+            </div>
+            <div class="categories-column-left">
+              <div class="index-list start-container">
+                <h2>Create</h2>
+                <div class="catalogue">
+                  <h3>
+                    <a href="command:ansible.lightspeed.playbookGeneration">
+                      <span class="codicon codicon-file-code"></span> Playbook with Ansible Lightspeed
+                    </a>
+                  </h3>
+                  <p>Create a lists of tasks that automatically execute for your specified inventory or groups of hosts.</p>
+                </div>
+                <div class="catalogue">
+                  <h3>
+                    <a href="command:ansible.content-creator.create-ansible-project">
+                    <span class="codicon codicon-file-zip"></span> Ansible playbook project
+                    </a>
+                  </h3>
+                  <p>Create a foundational framework and structure for setting your Ansible project with playbooks, roles, variables, templates, and other files.</p>
+                </div>
+                <div class="catalogue">
+                  <h3>
+                    <a href="command:ansible.content-creator.create-ansible-collection">
+                    <span class="codicon codicon-layers"></span> Ansible collection project
+                    </a>
+                  </h3>
+                  <p>Create a structure for your Ansible collection that includes modules, plugins, molecule scenarios and tests.
+                  </p>
+                </div>
+              </div>
+
+              <!-- <div class="index-list start-container">
+                <h2>Recent</h2>
+                <p>No recent activity</p>
+              </div> -->
+
+              <div class="index-list start-container">
+                <h2>Learn</h2>
+                <div class="catalogue">
+                  <h3>
+                    <a href="https://docs.ansible.com">
+                      Ansible documentation
+                      <span class="codicon codicon-link-external"></span>
+                    </a>
+                  </h3>
+                  <p>Explore Ansible documentation, examples and more.</p>
+                </div>
+                <div class="catalogue">
+                  <h3>
+                    <a href="https://docs.ansible.com/ansible/latest/getting_started/index.html">
+                      Learn Ansible development
+                      <span class="codicon codicon-link-external"></span>
+                    </a>
+                  </h3>
+                  <p>End to end course that will help you master automation development.</p>
+                </div>
+                <div class="catalogue">
+                  <h3>Once you are in the YAML file:</h3>
+                  <p>click Ctrl+L to fire the Ansible Lightspeed AI assistance for editing and explaining code.</p>
```

Comment:
Should this note be included on this page or inside one of the walkthroughs (or omitted?)

Reply:
This is something that I just copied over from the other file, no cleanup done as it was not related to the changes for this PR.